tests/llm/utils/test_mock_toolset.py

```
@@ -102,13 +102,13 @@ def test_sanitize_filename(self, input_filename, expected):
 
 
 def assert_toolset_enabled(mock_toolsets: MockToolsetManager, toolset_name: str):
-    for toolset in mock_toolsets.enabled_toolsets:
+    for toolset in mock_toolsets.toolsets:
         if toolset.name == toolset_name:
             assert (
                 toolset.status == ToolsetStatusEnum.ENABLED
             ), f"Expected toolset {toolset_name} to be enabled but it is disabled"
             return
-    assert False, f"Expected toolset {toolset_name} to be enabled but it missing from the list of enabled toolsets"
+    assert False, f"Expected toolset {toolset_name} to be found in the list of toolsets"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)